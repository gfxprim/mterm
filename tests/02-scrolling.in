6 4
a\n
b\n
c\n
d\n
e\n
Hello!
