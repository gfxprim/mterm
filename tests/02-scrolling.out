 ------
|d     |
|e     |
|Hello!|
|      |
 ------
size 4x6 cursor 3x0
